gui/Project.cxx

```diff
@@ -98,6 +99,11 @@ bool Project::read(QString const& path)
 
     return true;
   }
+  catch (kwiver::vital::config_block_exception e)
+  {
+    qWarning() << e.what(); // TODO dialog?
+    return false;
```

Comment:
we should keep the `// TODO set error` for us to remember to handle the error when the API exists.